packages/admin/src/pages/index.page.tsx

```diff
@@ -10,6 +11,7 @@ export const getServerSideProps = async ({
 }: GetServerSidePropsContext) => {
   const cookieValue = req.cookies[process.env.JWT_COOKIE_NAME!];
   const redirectUrl = query?.redirectUrl as string | undefined;
+  if (redirectUrl) validateRedirectUrl(redirectUrl);
```

Comment:
We probably ought to catch the error this throws and redirect to the login page.

Reply:
Good shout!